src/lib/table/cell.ts

```diff
@@ -89,7 +89,7 @@ export class MatColumnDef extends CdkColumnDef {
 })
 export class MatHeaderCell extends CdkHeaderCell {
   constructor(columnDef: CdkColumnDef,
-              elementRef: ElementRef) {
+              elementRef: ElementRef<HTMLElement>) {
```

Comment:
`<HTMLElement | HTMLTableCellElement>`

Reply:
Never mind, `type1 | type2` doesn't give you anything if `type2` extends `type1`.